test/ServerRequestTest.php

```diff
@@ -97,7 +97,19 @@ public function testRemovingAttributeReturnsCloneWithoutAttribute($request)
         $this->assertNull($new->getAttribute('foo', null));
     }
 
-    public function testUsesProvidedConstructorArguments()
+    public function provideMethods()
+    {
+        return array(
```

Comment:
Please avoid review comments about styling. If the CS tool not yield there is nothing to discuss.